v2/src/modules/block/components/MoreBlockDetails.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <v-card variant="flat" class="pa-4 pa-sm-6">
+        <template v-if="props.isLoading || !props.blockDetails">
+            <v-row class="my-2">
+                <v-col cols="12" md="6">
+                    <div class="skeleton-box rounded-xl mt-1" style="height: 24px"></div>
+                </v-col>
+            </v-row>
+            <div v-for="i in 5" :key="i" class="skeleton-box rounded-xl my-5" style="height: 24px"></div>
+        </template>
+        <template v-else>
+            <v-row>
+                <v-col lg="1">
+                    <div class="block-info mb-5">
+                        <p class="text-button mb-1">Size</p>
+                        <p class="text-info">
+                            {{ props.blockDetails.blockSize.detail }}
+                        </p>
+                    </div>
+                </v-col>
+                <v-col lg="1">
+                    <div class="block-info mb-5">
+                        <p class="text-button mb-1">Nonce</p>
+                        <p class="text-info">
+                            {{ props.blockDetails.nonce.detail }}
+                        </p>
+                    </div>
+                </v-col>
+                <v-col lg="1">
+                    <div class="block-info mb-5">
+                        <p class="text-button mb-1">Difficulty</p>
+                        <p class="text-info">
+                            {{ props.blockDetails.difficulty.detail }}
+                        </p>
+                    </div>
+                </v-col>
+                <v-col lg="3">
+                    <div class="block-info mb-5">
+                        <p class="text-button mb-1">Total Difficulty</p>
+                        <p class="text-info">
+                            {{ props.blockDetails.totalDifficulty.detail }}
+                        </p>
+                    </div>
+                </v-col>
+            </v-row>
+            <div class="block-info mb-5">
+                <p class="text-button mb-1">State Root</p>
+                <app-transform-hash is-blue :hash="props.blockDetails.stateRoot.detail" class="w-100" />
```

Comment:
you can actually remove is-blue  from all app-transform-hash components. they are not links